keys.go

```diff
@@ -81,6 +81,7 @@ type Key struct {
 	PurgeAllowedFrom    *time.Time  `json:"purgeAllowedFrom,omitempty"`
 	PurgeScheduledOn    *time.Time  `json:"purgeScheduledOn,omitempty"`
 	DualAuthDelete      *DualAuth   `json:"dualAuthDelete,omitempty"`
+	Search              string      `json:"sort,omitempty"`
```

Comment:
why it is `sort` here?

Reply:
Seems like typo issue here, will fix this